core/sound-supervisor/src/SoundAPI.ts

```diff
@@ -9,7 +9,12 @@ import { restartDevice, rebootDevice, shutdownDevice } from './utils'
 import { getSdk, BalenaSDK } from 'balena-sdk'
 import * as fs from 'fs'
 
-const VERSION = fs.readFileSync('VERSION', 'utf-8')
+let VERSION = '1.0.1';
+try {
+  VERSION = fs.readFileSync('VERSION', 'utf8').trim();
+} catch (err) {
+  console.warn('VERSION file not found. Using default version.');
+}
```

Comment:
Would recommend we use a ternary instead which makes it much cleaner.

```suggestion
const VERSION = fs.existsSync('VERSION') 
  ? fs.readFileSync('VERSION', 'utf8').trim() 
  : '1.0.0'; // or whatever your default version we decide```

Reply:
this seems much more elegant yes, this way the variable is locked after start and avoids any other issues.   We would delete my let in this scenario, just want to make sure we don't forget it.